htdocs/hrm/class/skill.class.php

```diff
@@ -417,7 +417,7 @@ public function fetchLines()
 			$this->error = $skilldet->error;
 			return $this->lines;
 		}
-		return -1;
+		return [];
```

Comment:
In Dolibarr, we have a convention. On error, we return -1 so we can make a difference between, the result was an empty array (nothing found) and there was a technical error.
So keep here the return -1 and instead change the doc comment from
`* @return array <0 if KO, array of skill level found`
into
`* @return array|int    <0 if KO, array of skill level found`